pkg/plugin/restore_pvc_action_plugin.go

```diff
@@ -61,7 +61,7 @@ func (p *NewPVCRestoreItemAction) Execute(input *velero.RestoreItemActionExecute
 	if blocked {
 		if crdName == "pods" {
 			return p.createPod(item)
-		} else if crdName == "images.imagecontroller.vmware.com" {
+		} else if utils.IsResourceBlockedOnRestore(crdName) {
 			// Skip the restore of image resources on Supervisor Cluster
 			return &velero.RestoreItemActionExecuteOutput{
 				SkipRestore: true,
```

Comment:
In line 70, if return error here restore will be PartiallyFailed when CRD is blocked, we may need to change this to return nil and add a log msg.

Reply:
Yes and no.

It might be better to add a line of log message to indicate what CRDs are skipped on restore.

But, for CRDs in `ResourcesToBlock` Map but not `ResourcesToBlockOnRestore` Map, we do need to return an error because those CRDs and their CRs should not even be included in the list of backup items in Velero backup.